stratum/hal/lib/barefoot/bfrt_packetio_manager.cc

```diff
@@ -115,9 +95,17 @@ ::util::Status BfrtPacketioManager::Shutdown() {
     packetout_header_.clear();
     packetin_header_size_ = 0;
     packetout_header_size_ = 0;
+    if (packet_receive_channel_) {
+      packet_receive_channel_->Close();
```

Comment:
The reason I add this check is that the `packet_receive_channel_` will be null if we push an invalid p4info

Reply:
https://github.com/stratum/stratum/blob/70d1a0c869634eac0bd3722101ce25c5e82d924c/stratum/hal/lib/bcm/bcm_chassis_manager.cc#L1481-L1486

Reply:
this is better, will do